apps/examples/fs_test/smartfs_test.c

```diff
@@ -0,0 +1,337 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/time.h>
+#include <fcntl.h>
+#include <errno.h>
+
+/****************************************************************************
+ * Definitions
+ ****************************************************************************/
+#define USAGE                                                           \
+	"\n FS R/W Performance Test\n"                                      \
+	"\n Usage :     smartfs_test <operation> <path> <size> <iteration> <print_option>\n"  \
+	"\nOperation :  1.Read/Write / 2.Read / 3.Write / 4. Write/Remove \n"                  \
+	" Path :        Test File Path\n"                                   \
+	" Size :        Write/Read Data size\n"                             \
+	" Iteration :   Number of repetitions\n"                            \
+	" Detail :      Enable(y) or Disable(n) Printing Details Option\n"  \
+	"               (Printing is affects at R/W Performance)\n"         \
```

Comment:
There are two verbs, `is` and `affects`.
I think `Printing affects ...` is better.